.github/workflows/ci-hotfix.yml

```diff
@@ -0,0 +1,36 @@
+name: Github-CI-Hotfix-Release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+env:
+  # secrets can be set in settings/secrets on github
+  DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+
+jobs:
+  deploy:
+    if: contains(github.event.base_ref, 'refs/heads/hotfix_v')
+    # only run on staging branch
+    name: deploy release
+    runs-on: ubuntu-latest
+    env:
+      FROM_TAG_PREFIX: hotfix-github
+      TO_TAG_PREFIX: release-github
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup docker
+        run: |
+          sudo ./ci/github/helpers/setup_docker_compose.bash
+          ./ci/github/helpers/setup_docker_experimental.bash
+          ./ci/github/helpers/setup_docker_buildx.bash
+          echo ::set-env name=DOCKER_BUILDX::1
+      - name: set owner variable
+        run: echo ::set-env name=OWNER::${GITHUB_REPOSITORY%/*}
+      - name: set git tag
+        run: echo ::set-env name=GIT_TAG::${GITHUB_REF##*/}
+      - name: deploy
+        run: ./ci/deploy/dockerhub-tag-version.bash
```

Comment:
not sure why this is here and for what purpose. I think the hotfix was working till now or is there something I don't know about? and it's a duplicate of the release workflow that will be also triggered...

Reply:
should I remove it? I also do not understand what it is doing here 😞

Reply:
removed it